socket.yaml

```diff
@@ -0,0 +1,6 @@
+# top level version field is required
+version: 2
+
+projectIgnorePaths:
+  - turborepo-tests
+  - packages/turbo-codemod/__tests__/
```

Comment:
We'll prob need more here, but this looks good for now and we can start extending it!